drivers/spi/Kconfig.nrfx

```diff
@@ -203,4 +203,7 @@ config SPI_NRFX_RAM_BUFFER_SIZE
 	  supplying buffers located in flash to the driver, otherwise such
 	  transfers will fail.
 
+config SPI_NRFX_HIGH_DRIVE_STRENGTH
+	bool "Use high drive strength for SCLK and MOSI pins"
```

Comment:
I think it should be configurable on a per-instance basis, so devicetree seems to be a more appropriate place for such option.